core/src/main/scala/io/prediction/core/CleanedDataSource.scala

```diff
@@ -116,11 +121,52 @@ trait CleanedDataSource {
   def cleanedPEvents(sc: SparkContext): RDD[Event] = {
     val stage1 = getCleanedPEvents(sc)
     val result = cleanPEvents(sc, stage1)
+    val originalEvents = PEventStore.find(appName)(sc)
     val (appId, channelId) = Common.appNameToId(appName, None)
-    PEventStore.wipe(result, appId, channelId)(sc)
+    wipe(result.collect.toSet, originalEvents.collect.toSet, appId, channelId)
+    //PEventStore.wipe(result, appId, channelId)(sc)
     result
```

Comment:
we should probably change the name now to something like a command because we'll have to live with it for awhile and for non-UR templates even